airbyte-cdk/python/unit_tests/sources/file_based/test_scenarios.py

```diff
@@ -81,11 +81,11 @@ def _verify_read_output(output: EntrypointOutput, scenario: TestScenario[Abstrac
 
     sorted_expected_records = sorted(
         filter(lambda e: "data" in e, expected_records),
-        key=lambda x: ",".join(f"{k}={v}" for k, v in sorted(x["data"].items(), key=lambda x: x[0]) if k != "emitted_at"),
+        key=lambda x: ",".join(f"{k}={v}" for k, v in sorted(x["data"].items(), key=lambda x: (x[0], x[1])) if k != "emitted_at"),
```

Comment:
can you rename the inner lambda x? 

`lambda x: (x[0], x[1])`
shadows
`key=lambda x:`

using a more descriptive name would also make it easier to understand what this one-liner does

Reply:
No problem, done.